privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -0,0 +1,208 @@
+
+## Header 
+
+logo-alt= { -brand-name-firefox-relay }
+nav-menu = Menu
+nav-home = Home
+
+# FAQ stands for Frequently Asked Questions. The intent of this page is to answer commonly asked questions.
+nav-faq = FAQ
+nav-profile-sign-in = Sign In
+nav-profile-manage-fxa = Manage your { -brand-name-firefox-account }
+nav-profile-sign-out = Sign Out
+nav-profile-sign-out-relay = Sign Out of { -brand-name-relay }
+nav-profile-sign-out-confirm = Are you sure you want to sign out?
+
+## Footer
+
+nav-footer-privacy = Privacy
+nav-footer-relay-terms = { -brand-name-relay } Terms
+nav-footer-legal = Legal
+
+## Bento Menu
+
+bento-button-title = { -brand-name-firefox } apps and services
+fx-makes-tech = { -brand-name-firefox } is tech that fights for your online privacy.
+made-by-mozilla = Made by { -brand-name-mozilla }
+fx-desktop = { -brand-name-firefox-browser } for Desktop
+fx-lockwise = { -brand-name-firefox-lockwise }
+fx-mobile = { -brand-name-firefox-browser } for Mobile
+fx-monitor = { -brand-name-firefox-monitor }
+fx-pocket = { -brand-name-pocket }
+mobileCloseBentoButtonTitle = Close menu
+
+## Home Page - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/
+
+home-hero-headline = Hide your real email address to help protect your identity
+home-hero-copy = { -brand-name-firefox-relay } makes it easy to create aliases, randomly generated email addresses that forward to your real inbox. Use it to protect your online accounts - and your identity - from hackers. Sign in with your { -brand-name-firefox-account } to get started.
+home-hero-cta = { nav-profile-sign-in }
+
+how-it-works-headline = How It Works
+how-it-works-subheadline = Protect your personal identity everywhere you use the { -brand-name-firefox-browser }.
+how-it-works-step-1-headline = Install the extension
+
+# Variables:
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+how-it-works-step-1-copy-html = <a href="{ $url }" { $attrs }>Download the { -brand-name-relay } extension for { -brand-name-firefox }</a>. Select the icon that appears on your { -brand-name-firefox } toolbar to access the sign in page. Sign in with your { -brand-name-firefox-account } to get started.
+how-it-works-step-2-headline = Create a new alias
+how-it-works-step-2-copy = As you browse, the { -brand-name-relay } icon will appear in form fields where sites ask for your email address. Select it to generate a new, random address that ends in @relay.firefox.com. { -brand-name-relay } will forward messages to the primary email address associated with your account.
+how-it-works-step-3-headline = Manage your account
+how-it-works-step-3-copy = Sign in to the { -brand-name-relay } website to keep track of the aliases you’ve created.  If you find that one receives spam or unwanted messages, you can block all messages or even delete the alias, right from the management page.
```

Comment:
Single space two sentences `...you’ve created. If you find...`

Reply:
I broke this line up into separate lines, and removed the extra space. Please let me know if my new line breaks are causing any issues.